src/index.html

```diff
@@ -17,11 +17,400 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link references a SCSS file ('styles/main.scss'). Browsers cannot directly interpret SCSS files. Ensure that this SCSS file is compiled to a CSS file and link to the resulting CSS file instead.